README.md

```diff
@@ -102,7 +102,10 @@ Steps 7 (Define) and 8 (Code) can occur concurrently. This may happen when you a
 
 ## The Process
 
-This process is composed of 8 steps which are introduced below:
+This process is composed of 8 steps which are introduced below.
+
+> A recommended talk about the process is ["Sociotechnical Architecture: co-designing technical & organizational architecture to maximize impact"](https://www.youtube.com/watch?v=ekMPm78KFj0&feature=youtu.be&t=1820) by Eduardo da Silva. He explains the process in the context of typical phases of designing sociotechnical architectures. Eduardo groups the activities in four phases "Align & Understand", "Strategic architecture", "Strategy & Org design" and "Tactical architecture".
```

Comment:
Really good entry (thank you for referring my talk :D). I think we could do a little bit of rewording to improve it even further. Something like: `A recommended talk that gives an overview of the process is .... Furthermore, Eduardo groups the activities of the process (and its 8 steps) in four distinct phases, namely: "1. Align & Understand", "2. Strategic Architecture", "3. Strategy & Org Design" and "4. Tactical Architecture".`

Reply:
Maybe you can also provide a link to the deck - directly to the pages where I give the overview. For example create a link on the "four distinct phases" text to https://speakerdeck.com/emgsilva/intro-to-sociotechnical-architecture-co-designing-technical-and-organizational-architecture-to-maximize-impact?slide=31